src/main/validation/UserValidation.ts

```diff
@@ -9,7 +9,7 @@ export class UserValidation {
     firstName: Joi.string().required(),
     lastName: Joi.string().required(),
     email: Joi.string().email().required(),
-    password: Joi.string().min(8).required(),
+    idFirebase: Joi.string().required(),
```

Comment:
Not sure if this would be the place to do it, but when the user is setting the password, we should probably restrict certain values (ie. 12345678) and set a character minimum

Reply:
I removed the validation for password since we use firebase server to store password, BE are supposed to not receive password anymore.

Reply:
Ahh gotcha! For the validation we have on the firebase server, what validation do we have?

Reply:
sorry what do you mean the validation on the firebase server? If this means the validation for `password` and `email` when we sign up/in, I guess the validation will be done in FE since we would sign up/in between FE and firebase.